packages/angular_devkit/core/package.json

```diff
@@ -7,6 +7,23 @@
   "keywords": [
     "core"
   ],
+  "exports": {
+    ".": {
+      "default": "./src/index.js",
+      "types": "./src/index.d.ts"
+    },
+    "./node": {
+      "default": "./node/index.js",
+      "types": "./node/index.d.ts"
+    },
+    "./node/testing": {
+      "default": "./node/testing/index.js",
+      "types": "./node/testing/index.d.ts"
+    },
+    "./package.json": "./package.json",
+    "./*": "./*.js",
+    "./*.js": "./*.js"
```

Comment:
Are these evaluated in order, or by longest prefix matching? In case of the former these need to be swapped.